app/views/policies/_form.html.slim

```diff
@@ -0,0 +1,28 @@
+.x_content
+  /! Smart Wizard
+  p Policy details
+  #wizard.form_wizard.wizard_horizontal
+    ul.wizard_steps.anchor
+      li
+        a.selected href="#step-1" isdone="1" rel="1" 
+          span.step_no 1
+          span.step_descr
+            | Step 1
+            br/
+            small Step 1 Policy Details
```

Comment:
@KaranBamaniya-BTC : First tab should display category, company , plan, start date, end date & other insurance related fields